src/views/AddItem.jsx

```diff
@@ -1,3 +1,101 @@
-export function AddItem() {
-	return <p>Hello from the <code>/add-item</code> page!</p>
+import { useState } from 'react';
+import { addItem } from '../api/firebase';
+
+export function AddItem({ listToken }) {
+	const [daysUntilNextPurchase, setTimeFrame] = useState('7');
+	const [itemName, setItem] = useState('');
+	const [itemData, setData] = useState({});
```

Comment:
Oh how interesting, to need to create separate state for itemName and itemData here. Great job digging into debugging to find this solution!

Reply:
I am curious about the state of daysUntilNextPurchase being a string of '7' instead of a number. Will that affect anything?

Reply:
I think you're onto something here Trevor, the daysUntilNextPurchase will eventually go into a mathmatical formula to predict when a user would buy that item next and it will need to be a number for those math operations

Reply:
Good catch Trevor, we had changed it to a number in the form but forgot to in the useState. Thank you for that, we'll change it!

Reply:
@Trevor-Rezac we actually need this number to be a string for the form to work correctly - show the first option selected by default and be able to select it again the user changes their mind. If it set to a number in the state, it fails to work.
What is stored as the value property goes into the formula calculating the next purchase date. @lindseyindev